release-content/0.14/migration-guides/12453_13277_14029_sorted_and_binned_render_phase_items.md

```diff
@@ -0,0 +1,34 @@
+Usage of `PhaseItem` has been split into `BinnedPhaseItem` and `SortedPhaseItem`. If you have custom `PhaseItem`s you will need to choose one of the new types. Notably some phases _must_ be Sorted (such as Transparent and Transmissive), while others can be Binned. Effectively Sorted is "what Bevy did before" and Binned is new, and the point of this change is to avoid sorting when possible for improved performance.
```

Comment:
```suggestion
Usage of `PhaseItem` has been split into `BinnedPhaseItem` and `SortedPhaseItem`. If you have custom `PhaseItem`s, you will need to choose one of the new types. Notably, some phases _must_ be sorted (such as transparent and transmissive), while others can be binned. Effectively, sorted is "what Bevy did before" while binned is a new method that places batchable items together, improving performance. Note that binned phase items largely remove z-fighting.
```

Just a few wording improvements that explains what binned phase items do, and de-capitalizes some words. (Did you have a specific reason for doing so?)

Reply:
I don't understand why binned phase item would remove z-fighting. The sentence seems to imply that z-fighting exists with sorted and is fixed with binned, which is quite unintuitive.

Reply:
I mentioned it because of something @mockersf wrote, though I have not experienced it myself. Perhaps it should be removed.

Reply:
reasoning behind capitalization was that Transparent has `Transparent2d`, `Transparent3d`, and `TransparentUi` and I was trying to convey that this is "A Real Thing" and not "a vague idea" without writing them all out. Not terribly important though.

Reply:
Maybe use "(such as  `Transparent2d`, `Transmissive3d`, _etc._)"?